packages/terra-button/src/variants/_AnchorButton.jsx

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
+import ThemeContext from 'terra-theme-context';
+import styles from '../Button.module.scss';
+import { ButtonTypes, ButtonVariants } from '../_constants';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Sets the href. Renders the component as an anchor tag.
+   */
+  href: PropTypes.string.isRequired,
+  /**
+   * Whether or not the button should display as a block.
+   */
+  isBlock: PropTypes.bool,
+  /**
+   * Whether or not the button has reduced padding
+   */
+  isCompact: PropTypes.bool,
+  /**
+   * Whether or not the button should be disabled.
+   */
+  isDisabled: PropTypes.bool,
+  /**
+   * Callback function triggered when mouse is pressed.
+   */
+  onMouseDown: PropTypes.func,
+  /**
+   * Callback function triggered when clicked.
+   */
+  onClick: PropTypes.func,
+  /**
+   * Callback function triggered when button loses focus.
+   */
+  onBlur: PropTypes.func,
+  /**
+   * Callback function triggered when button gains focus.
+   */
+  onFocus: PropTypes.func,
+  /**
+   * Callback function triggered when key is pressed.
+   */
+  onKeyDown: PropTypes.func,
+  /**
+   * Callback function triggered when key is released.
+   */
+  onKeyUp: PropTypes.func,
+  /**
+   * Callback ref to pass into the dom element.
+   */
+  refCallback: PropTypes.func,
+  /**
+   * Sets the button text.
+   */
+  text: PropTypes.string.isRequired,
+  /**
+   * Additional information to display as a native tooltip on hover. It will fallback to using `text` if not provided.
+   */
+  title: PropTypes.string,
```

Comment:
Have these two properties been vetted for a11y concerns? For example: how confident are you in the screen reader behavior when reading these two props?

Reply:
Screen reader was able to read out based on content passed for "text" prop. 
"title" would use "text" if not provided.
![25A14EE6-3571-4708-BE98-98F41A137C18_1_105_c](https://user-images.githubusercontent.com/40137430/186396794-76ef5bbd-c7f4-47e8-ad3a-951701aa7305.jpeg)